build/package/Dockerfile

```diff
@@ -14,8 +14,17 @@ RUN go build -o server-alpha ./cmd/server
 
 ## Serve only the compiled binary in the second stage
 FROM alpine:3.17.6 AS serve
+WORKDIR /go/app
```

Comment:
Wofür das hier? Würde das jetzt im Endeffekt nicht den späteren Copy-Befehl in /go/app/go/app setzen?